packages/node_modules/@ciscospark/widget-meet/src/messages.js

```diff
@@ -22,6 +22,22 @@ export default defineMessages({
     id: `ciscospark.container.meet.button.hangup`,
     defaultMessage: `Hangup`
   },
+  okButtonLabel: {
+    id: `ciscospark.container.meet.button.ok`,
+    defaultMessage: `Ok`
+  },
+  callErrorDeviceNotFound: {
+    id: `ciscospark.container.meet.message.deviceNotFound`,
+    defaultMessage: `No calling devices found`
+  },
+  callErrorNotAllowed: {
+    id: `ciscospark.container.meet.message.deviceNotFound`,
+    defaultMessage: `Media access has been blocked by browser`
```

Comment:
Can we change this to: `Cisco Spark cannot access your camera. Please check your browser settings and try again."